data/coalition/coalition jobs.txt

```diff
@@ -1322,3 +1322,432 @@ mission "Coalition: From Games 3"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		"supplied resistance" ++
+		outfit "Armageddon Core" -1
+		payment 9000000
+		dialog `Some Resistance members unload the Armageddon Core after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Core away.`
+
+
+mission "Resistance: Fusion Reactor Delivery"
+	name "Fusion Reactor Delivery"
+	description "Resistance members are offering <payment> for you to deliver a Fusion Reactor to <destination>"
+	repeat
+	minor
+	to offer
+		random < 7
+		has "pledged to resistance"
+		has "Resistance: Reactors: done"
+		not "Resistance: Fusion Reactor Delivery: active"
+		not "assisted heliarch"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch Containment 4-A: offered"
+	to fail
+		has "Heliarch Investigation 2: active"
+	source
+		government "Coalition"
+	destination
+		distance 3 8
+		not government "Heliarch"
+	on offer
+		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Fusion Reactor."`
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		"supplied resistance" ++
+		outfit "Fusion Reactor" -1
+		payment 5000000
+		dialog `Some Resistance members unload the Fusion Reactor after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Reactor away.`
+
+
+mission "Resistance: Breeder Reactor Delivery"
+	name "Breeder Reactor Delivery"
+	description "Resistance members are offering <payment> for you to deliver a Breeder Reactor to <destination>"
+	repeat
+	minor
+	to offer
+		random < 10
+		has "pledged to resistance"
+		has "Resistance: Reactors: done"
+		not "Resistance: Breeder Reactor Delivery: active"
+		not "assisted heliarch"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch Containment 4-A: offered"
+	to fail
+		has "Heliarch Investigation 2: active"
+	source
+		government "Coalition"
+	destination
+		distance 3 6
+		not government "Heliarch"
+	on offer
+		dialog `As you enter the spaceport, you're dragged away to an isolated area by someone who claims to be a Resistance member. They speak briefly, saying, "Comrades in <destination> have requested a Breeder Reactor."`
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		"supplied resistance" ++
+		outfit "Breeder Reactor" -1
+		payment 2500000
+		dialog `Some Resistance members unload the Breeder Reactor after you land on an isolated part of <planet>, as instructed. They hand you <payment> before quietly transporting the Reactor away.`
+
+
+mission "Resistance: Torpedo Delivery"
+	name "Torpedo Delivery"
+	description "Resistance members are offering <payment> for you to deliver a Torpedo Launcher, complete with ammunition, to <destination>."
```

Comment:
```suggestion
	description "Resistance members are offering <payment> for you to deliver a Torpedo Launcher, complete with 30 torpedoes, to <destination>."
```
It would be a good idea to clarify how much is needed for those buying to cargo.